src/resources/classes/controller.js

```diff
@@ -30,4 +30,51 @@ const createOne = async (req, res) => {
     }
 }
 
-module.exports = { getAll, createOne };
\ No newline at end of file
+const deleteOne = async (req, res) => {
+    const targetId = parseInt(req.params.id)
+    try {
+        const deleteOneClass = await prisma.class.delete({
+            where: {
+                id: targetId,
+            }
+        })
+        res.json({ message: "Delete successful" })
+    } catch (error) {
+        console.error({ error })
+    }
+}
+
+const updateOne = async (req, res) => {
+    try {
```

Comment:
This looks good, there's a lot of `req.body.something` going on though and they all have the same names as the data required. You could destruct req.body at the top:
```
const {
    className, classType, classStatus etc. etc.
} = req.body;
```

then in the Prisma section you can use those directly:

```
data: {
    className,
    classType,
    etc.
}
```